frontend/pulse3d/src/pages/users-info.js

```diff
@@ -0,0 +1,257 @@
+
+const PageContainer = styled.div`
+  width: 80%;
+`;
+
+const Pagination = styled.div`
+  width: 100%;
+  background-color: var(--dark-gray);
+  display: flex;
+  justify-content: right;
+  padding: 1rem 1rem;
+`;
+
+const Container = styled.div`
+  display: flex;
+  position: relative;
+  justify-content: start;
+  padding: 0% 3% 3% 3%;
+  flex-direction: column;
+  margin-top: 1rem;
+`;
+
+export default function UserInfo() {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [rows, setRows] = useState([]);
+  const [currentRows, setCurrentRows] = useState([]);
+  const [users, setUsers] = useState([]);
+  const [actionAlertVisible, setActionAlertVisible] = useState(false);
+  const [modalMessage, setModalMessage] = useState();
+  const [actionToPreform, setActionToPreform] = useState("");
+  const [confirm, setConfirm] = useState(false);
+  const [emailToDelete, setEmailToDelete] = useState("");
+
+  const deleteUser = async () => {
+    fetch(`https://curibio.com/user-actions/${emailToDelete}`, {
+      method: "DELETE",
+    });
+  };
+  const deactivateUser = async () => {
+    fetch(`https://curibio.com/user-actions/${emailToDelete}`, {
+      method: "PUT",
+      body: JSON.stringify({ action_type: "deactivate" }),
+    });
+  };
+
+  const userActionSelection = async (option, name, email) => {
+    if (option === 1) {
+      setModalMessage(`Are you sure you would like to deactivate ${name}?`);
+      setActionToPreform("deactivate");
+      setEmailToDelete(email);
+    } else if (option === 0) {
+      setModalMessage(`Are you sure you would like to delete ${name}?`);
+      setActionToPreform("delete");
+      setEmailToDelete(email);
+    }
+  };
+
+  const getAllUsers = async () => {
+    try {
+      const response = await fetch("https://curibio.com/users");
+      if (response && response.status === 200) {
+        const usersJson = await response.json();
+        const userList = [];
+        for (const user of usersJson) {
+          userList.push(user);
+        }
+        setUsers(userList);
+        setCurrentRows(userList.slice(0, 10));
+      }
+    } catch (e) {
+      console.log("ERROR fetching all users info");
+    }
+  };
+
+  useEffect(() => {
+    getAllUsers();
+  }, []);
+
+  useEffect(() => {
+    if (users) {
+      setRows([...users]);
+    }
+  }, [users]);
+
+  useEffect(() => {
+    try {
+      if (actionToPreform === "delete" && confirm) {
+        deleteUser();
+      } else if (actionToPreform === "deactivate" && confirm) {
+        deactivateUser();
+      }
+    } catch (e) {
+      console.log("ERROR on user action ");
+    }
+  }, [confirm]);
+
+  useEffect(() => {
+    setCurrentRows(
+      rows.slice(10 * (currentPage - 1), 10 * (currentPage - 1) + 10)
+    );
+  }, [currentPage]);
+
+  return (
+    <>
+      <ModalWidget
+        open={actionAlertVisible}
+        labels={[modalMessage]}
+        buttons={["yes", "no"]}
+        header={"attention"}
```

Comment:
I would capitalize the buttons and headers to match how the other modals are set up